addons/mail/static/tests/tours/mail_full_composer_test_tour.js

```diff
@@ -18,9 +23,28 @@ tour.register('mail/static/tests/tours/mail_full_composer_test_tour.js', {
     content: "Write something in composer",
     trigger: '.o_ComposerTextInput_textarea',
     run: 'text blahblah',
+}, {
+    content: "Add some file in composer",
+    trigger: '.o_Composer_buttonAttachment',
+    run: async() => {
```

Comment:
It doesn't really matter but we might want to keep the provided scope instead of the global scope. (`run` is bound to a specific value in tour manager, we just never need it).

```suggestion
    async run() {
```